source/Server/Ldap/LdapCredentialValidator.cs

```diff
@@ -46,39 +47,49 @@ public class LdapCredentialValidator : ILdapCredentialValidator
         public IResultFromExtension<IUser> ValidateCredentials(string username, string password, CancellationToken cancellationToken)
```

Comment:
This and the follow method are really similar.  Should probably extract the common functionality at some point.